lib/lhm/migrator.rb

```diff
@@ -205,5 +206,11 @@ def index_ddl(cols, unique = nil, index_name = nil)
     def order_column
       @options[:order_column] || @origin.pk
     end
+
+    def assert_valid_idx_name(index_name)
+      if index_name && !(index_name.is_a?(String) || index_name.is_a?(Symbol))
+        raise ArgumentError, "index_name must be a string or symbol"
```

Comment:
now that you are here, might as well make sure the index is smaller than the max size